src/test/java/ca/corefacility/bioinformatics/irida/service/impl/integration/AnnouncementServiceImplIT.java

```diff
@@ -65,7 +65,7 @@
     public void testCreateAnnouncementAsAdmin() {
         final Authentication auth = SecurityContextHolder.getContext().getAuthentication();
         final User user = userService.getUserByUsername(auth.getName());
-        Announcement an = new Announcement("This is a message", user);
+        Announcement an = new Announcement("This is a message", "This is a message", false, user);
```

Comment:
For this line and similar below. Change the first `"This is a message"` in `new Announcement` to `"This is a message title"` since that is what is being set for that argument

Reply:
Done! Please see f0780ca71fa8d9c34a5fec1207e5b6b770f58e24.